ReleaseHistory.md

```diff
@@ -1,4 +1,8 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
+## **v4.5.1 UNRELEASED
+* BUG: Report `WRN997_InvalidTarget` if the path of the scan target contains invalid characters.
+* BUG: When the path of the scan target contains URL Percent-encoding characters, `uri.GetFilePath` should return the original string.
```

Comment:
>When [](http://example.com/codeflow?start=7&length=4)

this is not a good release note, it describes the code fix.
 
how does this help a customer understand whether we have fixed a problem they have seen?

Reply:
Updated with symptom.